tt_eager/tensor/tensor_impl.hpp

```diff
@@ -799,9 +799,9 @@ inline void print_trailing_comma(std::ostream& ss, std::size_t index, std::size_
 template <typename T>
 inline void print_datum(std::ostream& ss, T datum) {
     if (std::is_integral<T>::value) {
-        ss << fmt::format("{:5}", datum);
+        ss << fmt::format(fmt::runtime("{:5}"), datum);
```

Comment:
This is not a runtime value.

Reply:
changed to use `std::` functions instead